arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/floorfilter/FloorFilterView.kt

```diff
@@ -0,0 +1,1071 @@
+                0, 0
+        ).apply {
+            try {
+                // button display attributes
+                maxDisplayLevels = getInt(R.styleable.FloorFilterView_maxDisplayLevels, DEFAULT_MAX_DISPLAY_LEVELS)
+                hideCloseButton = getBoolean(R.styleable.FloorFilterView_hideCloseButton, DEFAULT_HIDE_CLOSE_BUTTON)
+                hideSiteFacilityButton = getBoolean(R.styleable.FloorFilterView_hideSiteFacilityButton, DEFAULT_HIDE_SITE_FACILITY_BUTTON)
+                hideSiteSearch = getBoolean(R.styleable.FloorFilterView_hideSiteSearch, DEFAULT_HIDE_SITE_SEARCH)
+                hideFacilitySearch = getBoolean(R.styleable.FloorFilterView_hideFacilitySearch, DEFAULT_HIDE_FACILITY_SEARCH)
+                ButtonPosition.fromInt(getInt(R.styleable.FloorFilterView_closeButtonPosition, DEFAULT_CLOSE_BUTTON_POSITION.value))?.let {
+                    closeButtonPosition = it
+                }
+
+                // text size attribute
+                textSizeSp = getDimensionPixelSize(R.styleable.FloorFilterView_textSize, 0).pixelsToSp(scaledDensity)
+                if (textSizeSp <= 0) {
+                    textSizeSp = DEFAULT_TEXT_SIZE_SP
+                }
+
+                // color attributes
+                textColor = getColor(R.styleable.FloorFilterView_textColor, DEFAULT_TEXT_COLOR)
+                selectedTextColor = getColor(R.styleable.FloorFilterView_selectedTextColor, DEFAULT_SELECTED_TEXT_COLOR)
+                buttonBackgroundColor = getColor(R.styleable.FloorFilterView_buttonBackgroundColor, DEFAULT_BUTTON_BACKGROUND_COLOR)
+                selectedButtonBackgroundColor = getColor(R.styleable.FloorFilterView_selectedButtonBackgroundColor, DEFAULT_SELECTED_BUTTON_BACKGROUND_COLOR)
+                closeButtonBackgroundColor = getColor(R.styleable.FloorFilterView_closeButtonBackgroundColor, DEFAULT_CLOSE_BUTTON_BACKGROUND_COLOR)
+                searchBackgroundColor = getColor(R.styleable.FloorFilterView_searchBackgroundColor, DEFAULT_SEARCH_BACKGROUND_COLOR)
+
+                // size attributes
+                buttonHeightDp = getDimensionPixelSize(R.styleable.FloorFilterView_buttonHeight, 0).pixelsToDp(displayDensity)
+                buttonWidthDp = getDimensionPixelSize(R.styleable.FloorFilterView_buttonWidth, 0).pixelsToDp(displayDensity)
+                if (buttonHeightDp <= 0) {
+                    buttonHeightDp = DEFAULT_BUTTON_HEIGHT_DP
+                }
+                if (buttonWidthDp <= 0) {
+                    buttonWidthDp = DEFAULT_BUTTON_WIDTH_DP
+                }
+            } finally {
+                recycle()
+            }
+        }
+        init(context)
+    }
+
+    /**
+     * Initializes this FloorFilterView by inflating the layout and setting the [RecyclerView]
+     * adapter.
+     *
+     * @since 100.13.0
+     */
+    private var floorsRecyclerView: RecyclerView? = null
+    private var floorListCloseButton: ImageView? = null
```

Comment:
please put the properties before the constructors and init block.

_The contents of a class should go in the following order:
Property declarations and initializer blocks
Secondary constructors
Method declarations
Companion object_

https://kotlinlang.org/docs/coding-conventions.html#class-layout